termine/announce.go

```diff
@@ -42,10 +42,11 @@ func init() {
 func announceStammtisch(t *data.Termin) error {
 	maildraft := `Liebe Treffler,
 
-am kommenden Donnerstag ist wieder Stammtisch. Diesmal sind wir bei {{.Location}}.
+am kommenden Donnerstag ist wieder Stammtisch. Diesmal sind wir bei
+{{.Location}}.
```

Comment:
The interesting line we briefly talked about is [this one](https://github.com/nnev/website/pull/20/files?diff=unified#diff-0039b705f632a6bd78af8bb434d2b8e7R110). According to [this example](https://play.golang.org/p/mSLxd0G1Ag), it should work fine placing `{{.Location}}.` right after the `sind wir bei`.
However, doing so results in the message text being cut after the inserted location in the mail sent. The `.` after the `{{.Location}}` is also dropped, along with the rest of the message.
The `msg` that the `body.Writer(msg)` gets is still the full text. However, it is not written entirely for some reason. [According to the godoc, as far as I understand it, the Writer-interface is not allowed to drop text without returning an error.](https://godoc.org/io#Writer) However, here it seems to be doing that.

Any obvious explanation for that behavior? any obvious bug I have overseen?

This type of text addresses the issue and can be merged without problems, but I find this behavior of the code… rather unsatisfying, as long as I haven't understood what's going wrong here.